setup.cfg

```diff
@@ -24,6 +24,7 @@ install_requires =
     datadog
     python-dotenv
     fastapi
+    aiofiles
```

Comment:
why?
you missed the asgi file I guess

Reply:
That's used by the installer, that shouldn't be there